public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -586,39 +612,62 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
       [`panel-alert-state--${alertState}`]: alertState !== undefined,
     });
 
+    // for new panel header design
+    const containerClassNamesNewPanelChrome = classNames({
+      'panel-container--absolute': isSoloRoute(locationService.getLocation().pathname),
+      'panel-container--transparent': transparent,
+      'panel-container--no-title': this.hasOverlayHeader(),
+      [`panel-alert-state--${alertState}`]: alertState !== undefined,
+    });
+
     const title = panel.getDisplayTitle();
     const padding: PanelPadding = plugin.noPadding ? 'none' : 'md';
 
+    const titleItems = [
+      <PanelHeaderTitleItems key="title-items" alertState={alertState} data={data} panelId={panel.id} />,
+    ];
     if (config.featureToggles.newPanelChromeUI) {
       return (
-        <PanelChrome
-          width={width}
-          height={height}
-          padding={padding}
-          title={title}
-          loadingState={data.state}
-          status={{
-            message: errorMessage,
-            onClick: (e: React.SyntheticEvent) => this.onOpenErrorInspect(e, InspectTab.Error),
-          }}
+        <section
+          className={containerClassNamesNewPanelChrome}
+          aria-label={selectors.components.Panels.Panel.containerByTitle(panel.title)}
         >
-          {(innerWidth, innerHeight) => (
-            <>
-              <ErrorBoundary
-                dependencies={[data, plugin, panel.getOptions()]}
-                onError={this.onPanelError}
-                onRecover={this.onPanelErrorRecover}
-              >
-                {({ error }) => {
-                  if (error) {
-                    return null;
-                  }
-                  return this.renderPanelContent(innerWidth, innerHeight);
-                }}
-              </ErrorBoundary>
-            </>
-          )}
-        </PanelChrome>
+          <PanelChrome
+            width={width}
+            height={height}
+            title={title}
+            loadingState={data.state}
+            status={{
+              message: errorMessage,
+              onClick: (e: React.SyntheticEvent) => this.onOpenErrorInspect(e, InspectTab.Error),
+            }}
+            description={!!panel.description ? this.onShowPanelDescription : undefined}
+            links={panel.links && panel.links?.length > 0 ? this.onShowPanelLinks : undefined}
+            panelNotices={{
+              getPanelNotices: () => getPanelQueryNotices({ frames: data.series }),
+              onClick: (e: React.SyntheticEvent, tab: string) => this.onOpenInspector(e, tab),
+            }}
+            titleItems={titleItems}
```

Comment:
I checked with Patrick and because of the nature of PanelNotices (very edge case), it is not important to have this next to the title. I will move these inside `titleItems` and we can have a cleaner API.